llvm/test/Instrumentation/InstrProfiling/binary-correlate.ll

```diff
@@ -0,0 +1,11 @@
+; RUN: opt < %s -passes=instrprof -profile-correlate=binary -S | FileCheck %s
+
+; CHECK: @__profd_foo = private global { i64, i64, i64, ptr, ptr, i32, [2 x i16] } { i64 {{.*}}, i64 {{.*}}, i64 ptrtoint (ptr @__profc_foo to i64)
```

Comment:
Though the CounterPtr's type (3rd argument) is i64, not ptr, I found that when lowered to assembly, it is just the address of the counter label. I guess it's fine.